README.md

```diff
@@ -1,13 +1,99 @@
-# README
+# GitReps Community
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Installation and Usage Instructions
+
+## 1: Clone this repository
+
+```
+git clone https://github.com/GitRepsCommunity/platform.git
+```
+
+## 2: System Dependencies:
+To run this Rails project as intended you will need to install:
+- Ruby
+- Ruby on Rails
+- A Ruby version manager (optional but recommended)
+   - RVM
+   - chruby
+   - rbenv
+   - asdf
+
+## 3: HomeBrew installation:
+
+### Install Ruby:
+```
+brew install ruby
+```
+
+### Verifying Ruby version:
+Most Ruby projects will contain a .ruby-version file with the expected version of Ruby to run.
+You can change your version of Ruby using a version manager like RVM.
+
+### Installing a version manager for Ruby (RVM):
+As a first step install GPG keys used to verify installation package:
+```
+brew install gnupg 
+
+gpg --keyserver keyserver.ubuntu.com --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+```
+
+Install RVM (development version):
+```
+\curl -sSL https://get.rvm.io | bash
+```
 
-Things you may want to cover:
+View available versions of Ruby:
+```
+rvm list
+```
 
-* Ruby version
+Install specific version of Ruby (3.1.4 for this project as of 7/28/2023):
```

Comment:
If we wanted to avoid referencing a specific version here along with the date, we could point people to the [.ruby-version](https://github.com/GitRepsCommunity/platform/blob/main/.ruby-version) file.